website/docs/addon_blender_artist.md

```diff
@@ -22,90 +30,121 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 
 ## Working with AYON in Blender
 
-AYON is here to ease you the burden of working on project with lots of
-collaborators, worrying about naming, setting stuff, browsing through endless
-directories, loading and exporting and so on. To achieve that, AYON is using
-concept of being _"data driven"_. This means that what happens when publishing
-is influenced by data in scene. This can by slightly confusing so let's get to
-it with few examples.
-
+AYON streamlines collaborative projects, simplifying tasks like naming conventions, configurations, and file navigation. It employs a "data-driven" approach, meaning the data within a scene dictates the publishing process. To clarify, let's explore some examples.
 
 ## Setting scene data
 
-Blender settings concerning framerate, resolution and frame range are handled
-by AYON. If set correctly in Ftrack, Blender will automatically set the
-values for you.
+Blender settings concerning **frame rate**, **resolution** and **frame range** are fully **handled by AYON**.
+If set correctly on the AYON server or in Ftrack, Blender will automatically set the values for you.
+
+:::tip
+If for some reason those aren't set correctly, force resolution and frame range/fps any time using **Set Frame Range** and **Set Resolution** actions sitting in the AYON menu.
```

Comment:
```suggestion
You can force a specific resolution and frame range/fps at any time using **Set Frame Range** and **Set Resolution** actions sitting in the AYON menu.
```
(picky, but I feel like the "if for some reason..." bit might sounds like we do not fully trust our implementation here)